setup/helpers/perf_helper.rb

```diff
@@ -414,7 +414,7 @@ def classify_foss_nodes(host)
 
   def update_hiera_datadir_in_local_config
     config = YAML.load_file(@local_hiera_config_path)
-    config[:yaml][:datadir] = '/etc/puppetlabs/code/environments/production/hieradata/'
+    config['hierarchy'].reject! {|backend| backend['name'] =~ /Classifier/i }
```

Comment:
I don't know how this hiera.yaml is laid down but it contains the classifier as a data source.